plugins/uefi-dbx/fu-uefi-dbx-device.c

```diff
@@ -250,7 +279,7 @@ fu_uefi_dbx_device_init(FuUefiDbxDevice *self)
 	fu_device_set_name(FU_DEVICE(self), "UEFI dbx");
 	fu_device_set_summary(FU_DEVICE(self), "UEFI revocation database");
 	fu_device_build_vendor_id(FU_DEVICE(self), "UEFI", "Linux Foundation");
-	fu_device_add_protocol(FU_DEVICE(self), "org.uefi.dbx");
+	fu_device_add_protocol(FU_DEVICE(self), "org.uefi.dbx-isodate");
```

Comment:
As this is effectively released and managed by Microsoft I'm wondering if this is really `org.uefi` anymore.

Reply:
it's only managed by Microsoft if you enrolled the MS `db` -- if you enrol a third party one then you have to get the dbx from the same place. e.g. HP and some of our three digit customers.

Reply:
But the new date sfuff only make sense in the context of a Microsoft dbx

Reply:
So maybe "isodate" is a bit misleading; it can be any integer at all specified in the XML metadata. Maybe just something like `org.uefi.dbx2`?

Reply:
Yeah; that would be better.